mne/cov.py

```diff
@@ -1713,6 +1718,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = DEFAULTS['scalings']
```

Comment:
if this is sufficient, should we consider changing it in places where it is currently implemented as:

    _scalings = DEFAULTS['scalings']
    if isinstance(scalings, dict):
        _scalings.update(scalings)
    scalings = _scalings